integrations-and-supported-tools/fastai/scripts/requirements.txt

```diff
@@ -0,0 +1,3 @@
+fastai==2.3.1
+neptune-client==0.9.16
+neptune-fastai
```

Comment:
Could you add an explicit version?

Reply:
Perhaps we could even merge it to

neptune-client[fastai]==0.9.16 (or whatever is the latest)

Reply:
Sure

Reply:
Got it, could you please mention it on the product-and-dev and ask if you should open a jira ticket for it. 
It sounds like a bug(ish) thing